MemoryGame-1B/Views/NewGame.xaml

```diff
@@ -8,55 +8,98 @@
       d:DesignHeight="450" d:DesignWidth="800"
       Title="NewGame">
     <Grid>
+        <Grid.Background>
+            <ImageBrush ImageSource="background.jpg"/>
```

Comment:
Name is inconsistent please make it with a capital b also where is background.jpg located?